### ISSUE TITLE
Make CC or saving single messages in Mass mailing possible 

### ISSUE THREAD

**Issue by MaxPerl** [justingit/dada-mail#1191]
Hello Justin,

I use dadamail for e-mails to members of a club, among other things. Once a year, the members must be invited to a general meeting. For this, it would be important to save the individual emails with the invitations. Is it possible to send the individual e-mails to a specific e-mail address cc or bcc? Or can the individual mails be saved somehow (e.g. as an mbox?)

Many thanks in advance!

Best wishes,
Max

**Comment by justingit** [comment]
Is Stationary what you're looking for?

**Comment by MaxPerl** [comment]
I don't think so. I need a confirmation that I have actually sent the emails to the individual recipients at a specific time, which is also recognised by a court, for example.

The best would be, to have the E-Mails in my Sent E-Mail folder or as cc.

**Comment by justingit** [comment]
OH OK, you don't want to save the content of the email message, you want a copy of the EMAIL. There is no current way to save a receipt of delivery or the email message itself. Dada Mail tries very hard to keep most of the data anonymous actually for PII reasons. The closest thing would be the "open" tracking, but that doesn't track if a message in a mass mailing was sent exactly.

**Comment by MaxPerl** [comment]
Thank you for the clarification. In fact, it is also a very specific requirement that is probably not used very often for mass mailings. But perhaps the implementation of an additional CC and BCC header in New Draft Message would be useful for the future? This could solve the problem very easily...

**Comment by justingit** [comment]
LOL no, not happening :)
